pact3.0/02-coin-cap.repl

```diff
@@ -0,0 +1,58 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Example of using refactoring "enforce" statements into capabilities
+
+
+; --------------------------------------------------------------------------
+; Installs account schema module
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-data { "admin-keyset": ["product-dept" "dev-dept"] })
+
+(module acct-module ACCT-GOVERNANCE
```

Comment:
let's leave out the acct-governance stuff from this example and just use `(defcap GOVERNANCE () true)` in `hello-coin`